### PR TITLE
Randomizerxd-contrib-Remedy-On-Demand

### PR DESCRIPTION
**Pull request by randomizerxd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- improved test_module when there are too many remedy incidents
- 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by randomizerxd — COMMENTED** [review]
updated RemedyOnDemand.yml tests

File: Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml [hunk 1]
```diff
@@ -37,23 +35,24 @@ configuration:
   name: incidentType
   type: 13
   required: false
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
```
Comment (YuvHayun) [inline Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml:42]: Same question as the previous PR, why did you add this param? it should be added automatically when adding the `isfetch: true` key with a default value of 1.

File: Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.js [hunk 2]
```diff
@@ -182,7 +182,7 @@ var fetchIncidents = function(query, test_module=false) {
         url += '?q=' + encodeURIComponent(query);
     }
     if (test_module) {
-        url += '?limit=1'
+        url += '?limit=1&q=Submitter="' + params.credentials.identifier + '"';
```
Comment (YuvHayun) [inline Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.js:185]: Also, can you explain why this change was needed?

File: Packs/Remedy-On-Demand/ReleaseNotes/1_0_6.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Remedy On-Demand
+- improved implementation of the *test-module* command
```
Comment (YuvHayun) [inline Packs/Remedy-On-Demand/ReleaseNotes/1_0_6.md:3]:
```suggestion
- improved implementation of the ***test-module*** command.
```

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hi @randomizerxd, thx for your quick reply.
I left 1 small fix request, and also please add explanations regarding the parameter you added to the .yml file and the change you made in the .js file.

When you're done, make sure to refer to (this link)[https://xsoar.pan.dev/docs/contributing/marketplace#resubmit-a-content-pack] for a quick guide about how to upload your fixes.


Please don't hesitate to contact me with any questions, I'm available both here and via slack.
Thanks again!

File: Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml (hunk 1 above)
Comment (randomizerxd) [reply Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml:42]: I answered this question in the previous PR: "I did not add this param, seems like XSOAR automatically added it? I see this param is in the existing Remedy integration already, why does it show as being added?"
I can remove this param since it should be added automatically when adding the `isfetch: true` key

Comment (YuvHayun) [reply Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml]: I see you already removed it, thx!

File: Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.js (hunk 2 above)
Comment (YuvHayun) [reply Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.js:185]: Sorry, I missed your comments there.
Sounds good!

**Review by YuvHayun — APPROVED** [review]
